docs/resources/Application.md

```diff
@@ -93,11 +96,36 @@
 
 Returns the [application](#DOCS_RESOURCES_APPLICATION/application-object) object associated with the requesting bot user.
+
+## Edit Current Application % PATCH /applications/@me
+
+Edit properties of the app associated with the requesting bot user. Only properties that are passed will be updated. Returns the updated [application](#DOCS_RESOURCES_APPLICATION/application-object) object on success. 
+
+> info 
+> All parameters to this endpoint are optional
+
+###### JSON Params
+
+| Field                             | Type                                                                       | Description                                                                                                   |
+| --------------------------------- | -------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------- |
+| custom_install_url                | string                                                                     | Default custom authorization URL for the app, if enabled                                                      |
+| description                       | string                                                                     | Description of the app                                                                                        |
+| role_connections_verification_url | string                                                                     | Role connection verification URL for the app                                                                  |
+| install_params                    | [install params](#DOCS_RESOURCES_APPLICATION/install-params-object) object | Settings for the app's default in-app authorization link, if enabled                                          |
+| flags \*                          | integer                                                                    | App's public [flags](#DOCS_RESOURCES_APPLICATION/application-object-application-flags)                        |
+| icon                              | ?[image data](#DOCS_REFERENCE/image-data)                                  | Icon for the app                                                                                              |
+| cover_image                       | ?[image data](#DOCS_REFERENCE/image-data)                                  | Default rich presence invite cover image for the app                                                          |
+| interactions_endpoint_url \*\*    | string                                                                     | [Interactions endpoint URL](#DOCS_INTERACTIONS_RECEIVING_AND_RESPONDING/receiving-an-interaction) for the app |
+| tags                              | array of strings                                                           | List of tags describing the content and functionality of the app. Max of 5 tags.                              |
```

Comment:
Maybe this should also mention the max length of a tag? It appears to be `20`
![image](https://github.com/discord/discord-api-docs/assets/61027276/ed2e5eab-f6af-4b2b-a1f5-488fc8ea2552)